riscv/processor.cc

```diff
@@ -61,7 +61,7 @@ processor_t::processor_t(const isa_parser_t *isa, const cfg_t *cfg,
 
   disassembler = new disassembler_t(isa);
   for (auto e : isa->get_extensions())
-    register_extension(find_extension(e.c_str())());
+    register_extension(find_extension(e.c_str())(), true /* is_custom */);
```

Comment:
I think having separate `register_base_extension` and `register_custom_extension` is cleaner.

Reply:
Agreed. Changed.